server/models/UserModel/index.js

```diff
@@ -11,6 +11,8 @@ const userMongoProjection = {
 }
 
 const UserModel = {
+  all: async (db) =>
```

Comment:
Does every admin user have access to all users, including other admins, and should they include themselves in the result..?

Reply:
good question. This was migrating the query over from the routes file, so there should be no functional change.

```js
  const { appDb } = req.app.locals
  appDb
    .collection('users')
    .find({}, { _id: 0, configs: 0, member_of: 0, password: 0, last_logoff: 0 })
    .toArray(function (err, users) {
      if (err) {
        console.log(err)
        return res.status(502).send([])
      } else if (users.length == 0) {
        return res.status(404).send([])
      } else {
        return res.status(200).json(users)
      }
    })
```